scrooge-generator/src/main/resources/nodegen/service.mustache

```diff
@@ -92,10 +92,10 @@ export class Processor {
         this._handler = handler
     }
 
-    public process(input: TProtocol, output: TProtocol) {
+    public process(input: TProtocol, output: TProtocol, context?: any) {
```

Comment:
Yeah, I thought that was probably the case, but we'll bubble it up eventually. Just a thought.